### DISCUSSION TITLE
大家對說說哥/說說姊有什麼想法嗎？

### DISCUSSION THREAD

**Discussion by Jinyuu17** [1101-GS4719-C/Python-Community-GS4719-C#2157] [General]
其實身邊不乏有需多人答應你要幹啊，結果都不了了之，你們對它嗎有麼想法嗎？

舉一個例子，江蕙在2015年正式宣佈「封麥」，結束了她在台灣樂壇長達30多年的演藝生涯。作為台灣歌壇的「二姐」，她的歌聲深受廣大粉絲的喜愛，尤其是在台語流行音樂方面具有重要影響力。她在2015年舉行了「祝福」巡迴演唱會，這是她的告別演出系列，騙到了許多老粉絲不惜購買黃牛票也要入場看二姊罪一場演唱會。然而，江蕙在2024年國慶晚會上復出她於10月5日在台北大巨蛋獻唱，為國慶活動帶來了感動時刻。當晚，她演唱了經典歌曲《甲你攬牢牢》和《家後》，並以熟悉的歌聲再次震撼全場。這讓當時搶票去聽告別演唱會的我感覺被當韭菜割了，極度的不高興。

**Comment by Chattenoire** [comment]
說說哥說說姐通常是一個團體裡不可或缺，但是又讓人很無言的存在。

當然，若是以"反正他們就是說說人"為前提，氣氛能被這樣的發言方式活絡也是不錯的。
(雖然我都是回點點點的那個)

**Comment by Jinyuu17** [comment]
這讓我想到台灣的政壇，許多政客為了選票都在選舉期間開出了一堆華而不實的政見，當選後卻說窒礙難行。很多人會捍衛自己的權利，罷免當選人，但台灣因這樣被提出罷免的人少之又少。是不是台灣人都默許這樣的環境？

**Comment by 112121py** [comment]
我覺得有點像「幹部揪」的概念，阿每次都會叫一下，約了又說沒空，但我覺得有時候反而是一個互動的機會，像後面可能接阿這個好吃/好玩嗎之類的。阿不過說說怪這種就真的有點小煩，最近問朋友A要不要玩盤靈古城，他說好啊，但到現在都沒出現q，就會覺得忙沒關係，但可以不要答應rrr

**Comment by zhilin93517** [comment]
我有一個同學很喜歡亂揪，揪完之後問他阿不是要去，他又會回說 沒啦其實我沒有要去。我稱呼我的這位同學為說說怪，並不稱之為說說哥或說說姊，因為這樣叫的話很像在把他當兒童台的大哥哥大姊姊在叫。

**Comment by lai1008** [comment]
我國中時有一個同學很喜歡亂揪，揪完之後問他阿不是要去，他又會開玩笑地回說 沒啦其實我沒有要去。我稱呼他為說說怪，說了又不做。

**Comment by yuan3x2** [comment]
說說確實令人無語，但有時候也有有趣的一面，例如我跟朋友都會打賭說說怪這次是不是只是說說，還是他痛改前非決定實踐，增加生活趣味性，也可以用來當作挑侃的依據，此外有時候說說也算是一種釋放壓力的方法？說來爽這樣。

**Comment by zoo100130** [comment]
我都直接稱這種人說說怪，每次遇到這樣的人都覺得被浪費時間精力，後來看到這種人也漸漸習慣了，慢慢喪失對人的信任感和耐心。
下次我打算試試看又遇到說說怪用刷訊息攻擊和連環call魔法: 阿不是要揪!! 
破房的不能只有我一個: )

**Comment by Boying12** [comment]
個人是認為沒差，畢竟會這樣說通常都是很熟的好友之間的互動。在這樣的關係中，大家都了解彼此的底線和幽默感，這種輕鬆的交流方式實際上可以增進友誼。自己偶爾也會開玩笑地這樣說，因為在我看來，這種輕鬆的氛圍能讓人感到放鬆，減少生活中的壓力。我認為朋友之間的幽默感是非常重要的，它能夠讓我們在繁忙的生活中找到片刻的輕鬆，無需對每件小事都斤斤計較，讓生活變得更加有趣。
另外，過於認真對待每一件事情，可能會讓人感到疲憊和壓力，反而影響到人際關係的和諧。因此，我覺得在朋友之間，保持一種輕鬆的態度是非常必要的。無論是開玩笑還是分享日常生活中的趣事，這些都能讓我們的友情更加深厚，讓彼此在快樂中度過每一天。總之，無需小題大作，享受與朋友之間的輕鬆互動才是最重要的。

**Comment by qiu0109** [comment]
我覺得說說本來就是很熟的朋友之間聊天開玩笑而已，比如說不揪喔或下次一定，大部分的話很明顯就能聽出來真的就只是說說，我是覺得不需要太認真的對待，況且跟朋友聊天通常都是聊一些廢話，如果每句話都要叫真的話豈不是很累?

**Comment by christaaah** [comment]
我覺得要看當時的氣氛，如果當時的氣氛就是大家都在幹話，那說說就是一個炒熱氣氛的方法，但是已經做承諾了，或者說出這個決定的時候會影響他人心情的時候，我認為說說是會降低大家對他的信任的，因為會有一種被耍了的感覺。

**Comment by ShuliANN** [comment]
我覺得他們習慣對別人的計畫或邀請滿口答應，甚至表現得很熱情，但事後卻常常沒有任何行動或回應，最終不了了之。這樣的行為可能讓人覺得失望，甚至影響到彼此的信任感。

**Comment by ShuliANN** [comment]
但....他們為什麼會這樣？
有些人可能是習慣性不拒絕，怕拒絕會讓對方失望，所以就先答應再說，但實際上並沒有做好行動的準備。
也有可能是因為當下覺得這件事「很好玩」、「有興趣」，但後來冷靜下來後發現不合適，或者缺乏執行力。

**Comment by ShuliANN** [comment]
長此以往，我會認為他們的話沒有可信度，也沒有辦法將重要的事情交付給他們處理，因為你根本無法確定他們是否會真的付諸行動。
我覺得做人最基本的就是「言出必行」。如果不確定能做到，當下就應該直接拒絕，而不是一味地敷衍或為了當時的氣氛去答應。這不僅是對自己負責，也是對別人負責。我認為這樣的人其實是在消耗身邊人的耐心和信任，最終只會讓自己在別人眼中失去可靠的形象。

**Comment by Elaine4488** [comment]
以前的我對朋友間的每一句話都非常認真對待，但後來發現有些話只是當下隨口一說，並不一定有特別的意圖。慢慢地，我學會調整自己的心態，如果發現對方是這種個性，之後和他聊天時，我也會切換到輕鬆隨意的模式，當作是隨興亂聊即可。

**Comment by wcyeee** [comment]
我覺得我們平常聊天的說說對我個人來說就真的百分之98是說說，但對於像文章裡說到的這種封麥又打開來唱，如果我是你的話也會覺得自己是韭菜... 但可是我反過來站在歌星的角度，又有時候過了幾年之後又會想出來唱一下，完全不唱可能會很不舒服覺得很可惜。所以我自己個人是覺得，如果這位封麥歌星真的是有一段時間才出現的話，那我覺得還好，但如果這位歌星說要封麥，但又一直出來活動的話，那我真的會覺得不太行...

**Comment by 113607514** [comment]
對於「說說哥/說說姊」，我覺得每個人有不同的看法。有些人覺得他們的分享能帶來情感共鳴，簡單的語句往往能讓人感受到安慰和鼓勵；但也有一些人覺得這些人只是靠製造話題來吸引注意，內容空洞且過於矯情。無論如何，這些人物反映了現代社交媒體中，大家對情感表達和共鳴的一種需求。

**Comment by yue083139** [comment]
我覺得在很熟的朋友之間難免會有這種情況發生，我個人是沒那麼介意這種情況啦，畢竟多相處久了就會習慣這種人的行為模式了，有時候也確實不是故意的，只能說要告誡自己在約定說出口以前還是需要再三思考過後再說出來會比較好，不然可能會被不熟的人打上言而無信的標籤。
